### PR TITLE
FEAT: Add doc return data

### PR DESCRIPTION
**Pull request by mmoorepackt**
### Pull Request Type

- [x] ✨ feat
- [ ] 🐛 fix
- [ ] ♻️ refactor
- [ ] 💄 style
- [ ] 🔨 chore
- [ ] 📝 docs

### Relevant Issues

resolves #xxx

### What is in this change?

Describe the changes in this PR that are impactful to the repo.

### Additional Information

Add any other context about the Pull Request here that was not captured above.

### Developer Validations

- [ ] I ran `yarn lint` from the root of the repo & committed changes
- [ ] Relevant documentation has been updated
- [ ] I have tested my code functionality
- [ ] Docker build succeeds locally

### CODE REVIEW

File: server/endpoints/api/document/index.js [hunk 1]
```diff
@@ -49,9 +49,9 @@ function apiDocumentEndpoints(app) {
               error: null,
```
Comment (AStevensTaylor) [inline server/endpoints/api/document/index.js:49]: Missing Documentation for Swagger build

**Review by AStevensTaylor — CHANGES_REQUESTED** [review]
Missing the swagger documentation for the new `document` response

File: server/endpoints/api/document/index.js (hunk 1 above)
Comment (mmoorepackt) [reply server/endpoints/api/document/index.js:49]: Ah yes, good point!

**Review by AStevensTaylor — APPROVED** [review]
🚀 LGTM

**Pull request merged**